méli-mélo/th-empathy/style.css

```
@@ -0,0 +1,83 @@
+/* Thematic purple */
+.btn-empathy.btn-primary,
+.empathy.wb-steps.quiz .btn-primary {
+    background-color: #5C2790;
+}
+
+/* Purple button hover and focus state */
+.btn-empathy.btn-primary:hover,
+.btn-empathy.btn-primary:focus,
+.btn-empathy.btn-primary:active,
+.empathy.wb-steps.quiz .btn-primary:hover,
+.empathy.wb-steps.quiz .btn-primary:focus,
+.empathy.wb-steps.quiz .btn-primary:active {
+    background-color: #D6B9EC;
+    color: #5c2790;
+}
+
+/* Light Purple button default state */
+.btn-empathy.btn-default,
+.empathy.wb-steps.quiz .btn-default {
+    background-color: #FFF;
+    border-color: #5C2790;
+    color: #5C2790;
+}
+/* Light Purple button hover and focus state */
+.btn-empathy.btn-default:hover,
+.btn-empathy.btn-default:focus,
+.btn-empathy.btn-default:active,
+.empathy.wb-steps.quiz .btn-default:hover,
+.empathy.wb-steps.quiz .btn-default:focus,
+.empathy.wb-steps.quiz .btn-default:active  {
+    background-color: #D6B9EC;
+    border-color: #471F70;
+    color: #471F70;
+}
+
+/* Thematic Heading 1 */
+h1#wb-cont.empathy {
+    border-bottom: 0.18em solid #5C2790;
+    border-image: linear-gradient(to right, #5C2790 71px, transparent 71px);
+    border-image-slice: 1;
+}
+
+/*  Thematic Steps Quiz */
+
+/* Thematic progress bar customization */
+.empathy.wb-steps.quiz progress {
+    /* Reset the default browser styles */
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+    border: none;
+
+    background-color: #777;
+    color: #DED0F1;
+}
+
+/* Webkit Browsers - Chome, Opera and Safari */
+.empathy.wb-steps.quiz progress::-webkit-progress-bar {
+    background-color: #777;
+}
+.empathy.wb-steps.quiz progress::-webkit-progress-value {
+    background-color: #DED0F1;
+}
+
+/* Mozilla Firefox */
+.empathy.wb-steps.quiz progress::-moz-progress-bar {
+    background-color: #DED0F1;
+}
+
+/* Thematic radio buttons, check boxes and legend */
+.empathy.wb-steps.quiz .gc-chckbxrdio input[type="radio"]:checked + label::before {
+    background: #5c2790;
+}
+
+.empathy.wb-steps.quiz .gc-chckbxrdio input[type="checkbox"]:checked + label::after {
+    border-color: #5c2790;
+}
+
+.empathy.provisional.wb-steps.quiz .wb-tggle-fildst > legend.wb-steps-active {
+    color: #5c2790;;
+
+}
```

Please add an empty line at the end of this file.
```suggestion
}

```